2022-07_sprint/sprint_parties/media_kit.md

```diff
@@ -0,0 +1,56 @@
+| Data Umbrella  | [@DataUmbrella](https://twitter.com/DataUmbrella) |  [dataumbrella](https://www.linkedin.com/company/dataumbrella/) | [data.umbrella.dei](https://www.facebook.com/data.umbrella.dei) | [@data.umbrella](https://www.instagram.com/data.umbrella/)  | N/A | 
+| PyMC Dev  | [@DataUmbrella](https://twitter.com/pymc_devs) |  [pymc](https://www.linkedin.com/company/pymc/mycompany/) | N/A | [pymc.io](https://discourse.pymc.io)  |
+
+
+## Share Our Events
+
+Our event series can be shared in one or more of the following ways:  
+- Event website: https://pymc-data-umbrella.xyz/en/latest/
+- Event registration form:  https://tinyurl.com/du-pymc-os-ws
+- Tweet:  https://twitter.com/DataUmbrella/status/1538875289652142080
+- LinkedIn post: ​​https://www.linkedin.com/feed/update/urn:li:activity:6944647561630662656/
+- Social media card:  https://raw.githubusercontent.com/pymc-devs/pymc-data-umbrella/main/_static/banner_2022_07/2022_07_banner_1280×640px.png
+- Meetup event: can share links or cross-post the event with your Meetup group
+    - Pre-series Office Hours | Jul 2 (Saturday) | 13 UTC | 1 hour | [MeetUp](https://www.meetup.com/data-umbrella/events/286552154/)
+    - Working Session #1      | Jul 9 (Saturday) | 13 UTC | 3 hours| [MeetUp](https://www.meetup.com/data-umbrella/events/286552452/)
+    - Working Session #2      | Jul 22 (Friday)  | 16 UTC | 3 hours| [MeetUp](https://www.meetup.com/data-umbrella/events/286628677/)
+    - Working Session #3      | Aug 4/5 (Th/Fr)  | 23 UTC  | 3 hours | [MeetUp](https://www.meetup.com/data-umbrella/events/286628723/)
+    - Post-series Office Hours [^1] | Aug 18 (Sat) | 23 UTC| 1 hour  | [MeetUp](https://www.meetup.com/data-umbrella/events/286628791/)
+- Email copy (button to copy all text available at the top right of the code block)
+
+  ```none
+  We [insert org name] are excited to share that we are a Community Partner with Data Umbrella for their upcoming open source working sessions with the Python library PyMC.  PyMC is an open-source probabilistic programming language in Python.
+
+  This is a beginner-friendly event.  Skills that are helpful to have: Git, GitHub, Markdown, Sphinx, statistics, python, Bayesian statistics.
+
+  Additionally, we have a series of videos where participants can watch videos and add timestamps. There are lots of ways to participate and contribute to open source.
+
+  More information on joining the event:
+  a) Schedule of events: https://pymc-data-umbrella.xyz/en/latest/2022-07_sprint/schedule.html
+  b) Complete this registration form: https://tinyurl.com/du-pymc-os-ws
+  ```
+### Channels to Share
+- Communication servers: Slack, Discord
+- Social media: Facebook, Instagram
+- Mobile communities: WhatsApp, Telegram
+- Newsletters
+
+## Logos
+- Data Umbrella: [Brand and Logo](https://github.com/data-umbrella/info)
+- PyMC Devs:  [Brand & Logo](https://github.com/pymc-devs/brand)
+
+### Acknowledgment
```

Comment:
This is now a subsection inside logos. I'd either make it a level 2 heading or a non heading, we could try something like

```
---
**Acknowledgement**

...
```

that would add a horizontal line that creates a visual separation and use bold face to avoid having the section in the right sidebar toc

Reply:
fixed